### ISSUE TITLE
The variable material of Animal has not been assigned.

### ISSUE THREAD

**Issue by jaffourt** [SebLague/Ecosystem-2#8]
Not sure what is causing this error, or how to "assign the material variable of the Animal script in the inspector".

Any one?

`UnassignedReferenceException: The variable material of Animal has not been assigned.`
`You probably need to assign the material variable of the Animal script in the inspector.`
`UnityEngine.Material.SetColorImpl (System.Int32 name, UnityEngine.Color value) <0x14376a580 + 0x0006a> in <dd49aecee05944db91bf4e1da96114e0>:0`
`UnityEngine.Material.SetColor (System.String name, UnityEngine.Color value) (at /Users/builduser/buildslave/unity/build/Runtime/Export/Shaders/Shader.cs:490)`
`UnityEngine.Material.set_color (UnityEngine.Color value) (at /Users/builduser/buildslave/unity/build/Runtime/Export/Shaders/Shader.bindings.cs:135)`
`Animal.Init (Coord coord) (at Assets/Scripts/Behaviour/Animal.cs:62)`
`Environment.SpawnInitialPopulations () (at Assets/Scripts/Environment/Environment.cs:362)`
`Environment.Start () (at Assets/Scripts/Environment/Environment.cs:47)`

**Comment by Elijer** [comment]
@jaffourt So at the top level of the prefabs directory, _each_ prefab (bunny, fox, plant, tree) each has an unassigned "material" property. In that same directory, there is a "materials" folder. I dragged materials from that folder to that empty property. I didn't really know what I was doing, so I just used the two materials available in that folder willy nilly.

This was after I went into each different subdirectory for bunny, fox, etc., and assigned the 4 material slots that seems to exist for them. Even the object mysteriously named "tex". So. I kinda just assigned all the materials and eventually the errors went away and everything ran. Hope that works!
